package.json

```diff
@@ -13,17 +13,22 @@
   "dependencies": {
     "@emotion/react": "^11.13.0",
     "@emotion/styled": "^11.13.0",
+    "@hookform/resolvers": "^3.9.0",
     "@mui/icons-material": "^5.16.6",
     "@mui/material": "^5.16.6",
+    "@mui/styled-engine-sc": "6.0.0-alpha.18",
     "@mui/x-date-pickers": "^7.12.0",
     "@reduxjs/toolkit": "^2.2.7",
     "dayjs": "^1.11.12",
     "json-server": "1.0.0-beta.1",
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-hook-form": "^7.52.2",
     "react-redux": "^9.1.2",
-    "react-router-dom": "^6.26.0"
+    "react-router-dom": "^6.26.0",
+    "styled-components": "^6.1.12",
+    "yup": "^1.4.0"
```

Comment:
what's this.

Reply:
These libraries are mandatory for creating a combined create/update form, in our case, the appointment form.
The styled components are used to style our custom components, which let you write CSS in JavaScript.

Here's an [example of using the React Hook Form library](https://jasonwatmore.com/post/2020/10/14/react-hook-form-combined-add-edit-create-update-form-example)

Reply:
I meant to only highlight "yup" sorry

Reply:
yup it's just yup ☠️